spec/models/post_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe Post, type: :model do
+  describe 'is expected to have db columns' do
+    it { is_expected.to have_db_column :track_name }
+    it { is_expected.to have_db_column :artists }
+    it { is_expected.to have_db_column :image }
+    it { is_expected.to have_db_column :preview_url }
+    it { is_expected.to have_db_column :description }
+  end
+
+  describe 'Factory Bot' do
```

Comment:
why don't you have "is expected to have validation"?

Reply:
Because we have no validations yet